python-checks/src/test/resources/checks/dictionaryDuplicateKey.py

```diff
@@ -0,0 +1,71 @@
+def strings():
+  {"one": 1, "two": 2}
+  {"one": 1, "two": 2, "one": 3}  # Noncompliant {{Change or remove duplicates of this key.}}
+  #^^^^^               ^^^^^<
+  {"one": 1, "two": 2, 'one': 3}  # Noncompliant
+  {"""multi
+  line""": 1, # Noncompliant@-1
+  "two": 2,
+  """multi
+  line""": 3}
+
+  # We only raise if the strings have same values & prefixes, even if they would eventually evaluate to the same value
+  {"one": 1, "two": 2, u"one": 3}  # OK
+  {"one": 1, "two": 2, r"one": 3}  # OK
+  {"on" "e": 1, "two": 2, "o" "ne": 2} # Noncompliant
+  {"on" r"e": 1, "two": 2, "on" r"e": 2} # Noncompliant
+  {"on" r"e": 1, "two": 2, "o" r"ne": 2} # OK
```

Comment:
Python interpreter gives `{'one': 2, 'two': 2}` here, so that's a false negative. Maybe it should be marked explicitly as an accepted FN instead of just OK?